### DISCUSSION TITLE
hello bnyro

### DISCUSSION THREAD

**Discussion by surabbar** [libre-tube/LibreTube#1001] [General]
WB

**Comment by Bnyro** [comment]
Heyy :pp

**Reply by surabbar** [reply]
Nothing can top this community bro:pppppp

**Reply by Bnyro** [reply]
Why? :p

**Reply by surabbar** [reply]
by the irony ;)

**Reply by Bnyro** [reply]
Switch to Matrix maybe? :D

**Reply by Bnyro** [reply]
I mean you, not because of the community xD

**Comment by Bnyro** [comment]
I've never been away lol

**Reply by NeeRaj-2401** [reply]
Hey @Bnyro how r u

**Reply by Bnyro** [reply]
I'm fine, thanks, what are we doing here?

**Reply by NeeRaj-2401** [reply]
Nothing :) i noticed today, u  deleted ur telegram account i  guess.. so i was asking about your well being

**Reply by Bnyro** [reply]
> Nothing :) i noticed today, u  deleted ur telegram account i  guess.. so i was asking about your well being
>
I've actually created a new one a month ago, but I'm not that active on the LT channels

**Reply by NeeRaj-2401** [reply]
- Oh ohk, in case if u dont remember me 😅 we used to discuss bugs on telegram DM and u helped me to fix sponsor block switches

**Reply by Bnyro** [reply]
> - Oh ohk, in case if u dont remember me 😅 we used to discuss bugs on telegram DM and u helped me to fix sponsor block switches 
>
Oh, right, now I remember :)
Feel free to join the LibreTube Matrix room, that's now what Telegram used to be earlier ;)

**Reply by NeeRaj-2401** [reply]
Ok tc, see u there soon
